share/smack/include/smack.h

```diff
@@ -24,6 +24,10 @@ void __SMACK_top_decl(const char *fmt, ...);
 // with an integer argument (DSA gets confused otherwise)
 __attribute__((always_inline)) void __SMACK_dummy(int v);
 
+#ifdef SVCOMP
```

Comment:
I don't particularly like this solution. Ideally, SVCOMP and SMACK benchmarks should be compatible. I emailed SVCOMP mailing list to ask about this.